cantabular/dataset_list.go

```diff
@@ -6,16 +6,16 @@ import (
 	"github.com/shurcooL/graphql"
 )
 
-type PopulationTypeQueryDataset struct {
+type DatasetListItem struct {
 	Name graphql.String
 }
 
-type PopulationTypeQuery struct {
-	Datasets []PopulationTypeQueryDataset
+type DatasetListQuery struct {
```

Comment:
Just one small thing.. Could we keep more precise naming of the related objects.. i.e. if the query is called ListDatasets name the query, items tests etc ListDatasetsxxx rather than DatasetListxxx